src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -4094,6 +4094,18 @@ public void getVersionFiles() throws IOException, InterruptedException {
         fileMetadatasCount = getVersionFilesResponseEmbargoedThenRestricted.jsonPath().getList("data").size();
         assertEquals(1, fileMetadatasCount);
 
+
+        // Test Access Status Retention
+        UtilIT.setSetting(SettingsServiceBean.Key.MinRetentionDurationInMonths, "-1");
+        String retentionEndDate = LocalDate.now().plusMonths(240).format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+
+        // Create retention for test file 2 (Retention and Public)
+        Response createFileRetentionResponse = UtilIT.createFileRetention(datasetId, Integer.parseInt(testFileId2), retentionEndDate, apiToken);
+        createFileRetentionResponse.then().assertThat()
+                .statusCode(OK.getStatusCode());
+
+        // TODO something with the search criteria...  retentionExpired, no just public because retention has not expired ?
```

Comment:
I think that makes sense - but I see you haven't added any retention related values the FileAccessStatus enum. Perhaps one new RetentionPeriodExpiredvalues/text could be added? If that's done, then adding an unexpired restriction would still show the file in public (or other category) and a Retention expired one would make it unavailable/listed with that filter.

Reply:
Not sure how to add to that FileaccessStatus; RetentionPeriodExpired makes sense and the fact that is was public or not is not very relevant I think. For the not-expired we might end up with all possible combinations with public,restricted and embargo.

Reply:
Just checked that I do have an 'Acces' filter with 'Retention Expired (1)' at the left side when I make a file retention period expire. If I select it I get that one file with the search breadcrumb: 'Access: Retention Expired' above the results. This is without that FleAccessStatus enum.

Reply:
I am tempted to also add 'RetentionNotExpired' resulting in only those that have a retention set, but that has not expired. Would be useful in the Integration Test, or when you want to know which ones are near to expiring if you sort the result.